source/providerdoall.c

```diff
@@ -23,11 +23,12 @@
 #include <openssl/provider.h>
 #include "perflib/perflib.h"
 
-#define NUM_CALLS_PER_TEST         100000
+#define RUN_TIME 5
 
 size_t num_calls;
 static int err = 0;
-OSSL_TIME *times;
+size_t *counts;
+OSSL_TIME max_time;
 
 static int doit(OSSL_PROVIDER *provider, void *vcount)
```

Comment:
provider is unused here

Reply:
`OSSL_PROVIDER_do_all` requires `doit` to have those parameters